package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@auth0/auth0-spa-js": "^1.2.4",
     "@babel/register": "^7.6.2",
+    "@next/bundle-analyzer": "^9.2.1",
```

Comment:
what's this?

Reply:
Analyzes package sizes

Reply:
We can remove thought it might be useful